tests/Feature/Http/Controllers/ProductControllerTest.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+namespace Tests\Feature\Http\Controllers;
+
+use App\Http\Controllers\ProductController;
+use App\Http\Requests\ProductRequest;
+use Tests\TestCase;
+use App\Models\Product;
+use Illuminate\Http\UploadedFile;
+use Illuminate\Support\Facades\Storage;
+use Illuminate\Foundation\Testing\WithFaker;
+use JMac\Testing\Traits\AdditionalAssertions;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+
+class ProductControllerTest extends TestCase
+{
+    use RefreshDatabase, WithFaker, AdditionalAssertions;
```

Comment:
Got `AdditionalAssertions` do not exist. Had to install it via composer and then everything worked as it should. Do you have these requirements in composer?